server/handlers/project_command_output_handler.go

```diff
@@ -22,7 +23,8 @@ type AsyncProjectCommandOutputHandler struct {
 	projectStatusUpdater   ProjectStatusUpdater
 	projectJobURLGenerator ProjectJobURLGenerator
 
-	logger logging.SimpleLogging
+	logger        logging.SimpleLogging
+	numChansGauge stats.Gauge
```

Comment:
Let's move the stats collection into a decorator. Like we do for project command runner https://github.com/lyft/atlantis/blob/release-v0.17.1-lyft.1/server/events/instrumented_project_command_runner.go

Reply:
Done!